frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayDownloadReport.java

```diff
@@ -1,5 +1,7 @@
 /*
- * Copyright Siemens AG, 2013-2015. Part of the SW360 Portal Project.
+ * Copyright Siemens AG, 2013-2017. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
```

Comment:
this will cause merge conflicts with https://github.com/sw360/sw360portal/pull/482